packages/core/src/App/Components/Layout/Header/wallets/account-info-wallets.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { CSSTransition } from 'react-transition-group';
+import { Icon, WalletIcon, Text, AppLinkedWithWalletIcon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { formatMoney, getCurrencyDisplayCode } from '@deriv/shared';
+import { useStore, observer } from '@deriv/stores';
+import { getActiveLoginIDFromLocalStorage } from '@deriv/utils';
+import { useStoreWalletAccountsList, useStoreLinkedWalletsAccounts } from '@deriv/hooks';
+import { AccountSwitcherWallet, AccountSwitcherWalletMobile } from 'App/Containers/AccountSwitcherWallet';
+import { AccountsInfoLoader } from '../Components/Preloader';
+import AccountInfoWrapper from '../account-info-wrapper';
+import WalletBadge from './wallet-badge';
+
+type TAccountInfoWallets = {
+    toggleDialog: () => void;
+    is_dialog_on: boolean;
+};
+
+type TDropdownArrow = {
+    is_disabled?: boolean;
+};
+
+type TBalanceLabel = {
+    balance: number;
+    currency: string;
+    is_virtual: boolean;
+    display_code: string;
+};
+
+type TMobileInfoIcon = {
+    wallet_account: ReturnType<typeof useStoreWalletAccountsList>['data'][number];
+};
+
+type TDesktopInfoIcons = {
+    wallet_account: ReturnType<typeof useStoreWalletAccountsList>['data'][number];
+};
```

Comment:
Types are the same, can combine 
```suggestion
type TInfoIcon = {
    wallet_account: ReturnType<typeof useStoreWalletAccountsList>['data'][number];
};
```

Reply:
Updated.